cluster/examples/kubernetes/ceph/object-bucket-claim-retain.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: objectbucket.io/v1alpha1
+kind: ObjectBucketClaim
+metadata:
+  name: ceph-retain-bucket
```

Comment:
Bucket reclaimPolicy is defined in the storage class. For testing reasons there is one storage class with a reclaimPolicy of "Delete" and one with a reclaimPolicy of "Retain". And one OBC per storage class. Easy to delete one of the storage class+OBC pairs. Do you care which one?

Reply:
It's good to have both storage classes to show their usage. I was thinking both storage classes could just point to the same OBC in the examples since there is no difference. But I see you are just making it easier to use the different storage class examples. Maybe a comment would help the user recognize they shouldn't expect a difference in the yamls if you want to keep them.

Reply:
I see the confusion now: I had a typo in one of the OBC yamls. I've fixed that so that the two OBCs have different k8s names and reference different storage classes.
Summary: there are 2 storage classes, 1 to represent retaining the bucket after the OBC is deleted and the other deletes the bucket after its OBC is deleted. Admins create the SCs. Then there are 2 OBC yamls, 1 references the "delete" SC and the other references the "retain" SC.